src/Analyzers/MSTest.Analyzers/Helpers/DiagnosticIds.cs

```diff
@@ -35,4 +35,5 @@ internal static class DiagnosticIds
     public const string PublicMethodShouldBeTestMethodRuleId = "MSTEST0029";
     public const string TypeContainingTestMethodShouldBeATestClassRuleId = "MSTEST0030";
     public const string DoNotUseSystemDescriptionAttributeRuleId = "MSTEST0031";
+    public const string PreferNotUsingAlwaysTrueAssertConditionsAnalyzerRuleId = "MSTEST0032";
```

Comment:
Maybe something like `ReviewAlwaysTrueAssertConditionAnalyzerRuleId`, please also update the class and resx entries accordingly.